test/dotnet-new.Tests/Approvals/DotnetNewHelpTests.CannotShowHelpForTemplate_WhenAmbiguousLanguageChoice.verified.txt

```diff
@@ -1,6 +1,6 @@
 ﻿Unable to resolve the template, these templates matched your input:
 Template Name  Short Name  Language  Tags      
 -------------  ----------  --------  ----------
-Basic FSharp   basic       F#,VB     Test Asset
+Basic VB       basic       F#,VB     Test Asset
```

Comment:
@v-wuzhai - have you verified this change is acceptable? I don't know why updating to a new `dotnet/runtime` version would have caused this to change. My suspicion is that a change in the runtime is causing the results to come in a different order. Looking at the diff: https://github.com/dotnet/runtime/compare/bcc1d3d6f00fbcea3f454b2e35bceeaa51e604b1...0f70e9aa5f08847680162a6dc712e069f0334b52

https://github.com/dotnet/runtime/pull/97483 stands out as a possibility.

Reply:
I logged https://github.com/dotnet/runtime/issues/97668 for the breaks in the dotnet new tests. @stephentoub has a revert PR for the change we believe caused this - https://github.com/dotnet/runtime/pull/97666. That should unblock this flow.